charts/jobset/README.md

```diff
@@ -0,0 +1,81 @@
+# JobSet Helm Chart
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.1.0](https://img.shields.io/badge/AppVersion-0.1.0-informational?style=flat-square)
+
+This Helm chart installs the JobSet Controller in your Kubernetes cluster. JobSet is a Kubernetes controller that manages groups of related Jobs as a single unit.
+
+## Installing the Chart
+
+To install the chart with the release name `jobset`:
+
+```bash
+helm install jobset ./charts/jobset
+```
+
+## Configuration
+
+| Key                                       | Type   | Default                                | Description |
+|-------------------------------------------|--------|----------------------------------------|-------------|
+| affinity                                  | object | `{}`                                   |             |
+| certManager.certificate.duration          | string | `"8760h"`                              |             |
+| certManager.certificate.renewBefore       | string | `"720h"`                               |             |
+| certManager.enabled                       | bool   | `true`                                 |             |
+| crds.enabled                              | bool   | `true`                                 |             |
+| crds.install                              | bool   | `true`                                 |             |
+| fullnameOverride                          | string | `""`                                   |             |
+| image.pullPolicy                          | string | `"IfNotPresent"`                       |             |
+| image.repository                          | string | `"jobset-controller"`                  |             |
+| image.tag                                 | string | `""`                                   |             |
+| imagePullSecrets                          | list   | `[]`                                   |             |
+| kubeRbacProxy.image.pullPolicy            | string | `"IfNotPresent"`                       |             |
```

Comment:
We dropped kubeRbacPolicy and it will be dropped in 0.8.0. Let's focus on that for now.